packages/vue/tests/render-and-events.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
+
+test('test app has loaded', async ({ page }) => {
+  await page.goto('http://localhost:5173/');
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle(/Vue app for E2E tests/);
+
+  // Topic menu has rendered
+  const topicMenuButton = await page.locator('gcds-topic-menu button');
+  await expect(topicMenuButton).toHaveText('Main Menu');
+
+  // Breadcrumbs has rendered has rendered
+  const breadcrumbsLink = await page.locator('gcds-breadcrumbs-item');
+  await expect(breadcrumbsLink).toHaveText('Canada.ca');
+
+  // Footer has rendered
+  const footerLinks = await page.locator('gcds-footer a').count();
+  await expect(footerLinks).toBe(26);
+});
+
+test('select - events', async ({ page }) => {
```

Comment:
Could use a better description - open to ideas
```suggestion
test('Select component event handling', async ({ page }) => {
```